py/server/deephaven/experimental/iceberg.py

```diff
@@ -346,11 +346,13 @@ def with_id(self, sort_order_id: int) -> 'SortOrderProvider':
 
     def with_fail_on_unmapped(self, fail_on_unmapped: bool) -> 'SortOrderProvider':
         """
-        Returns a sort order provider that will fail if the sort order cannot be applied to the tables being written.
-        By default, if the sort order cannot be applied, the tables will be written without sorting.
+        Returns a sort order provider that uses the provided argument value to determine whether to fail or not if the
+        sort order cannot be applied to the tables being written. By default, all providers fail if the sort order
```

Comment:
Run the first sentence through AI.  It may have an easier to read suggestion, but otherwise good.